### DISCUSSION TITLE
Mac os x64 download is a windows .exe file

### DISCUSSION THREAD

**Discussion by dhomi** [jjazzboss/JJazzLab#457] [Q&A] [answered]
Can you please make the mac os x64.zip to be a mac app please. When I download the [JJazzLab-4.1.0-mac-x64.zip] I find the windows zip file with the jjazzlab.exe on it. The jjazzlab binary file does not work on mac os propperly, it refuses to use the jdkhome parameter also.
The [JJazzLab-4.1.0-mac-aarch64.pkg] would be fine if it was for x64. I use Mac Monterey, 12.7.5

Help would be appreciated

**Comment by jjazzboss** [comment]
What error messages do you get when you run bin/jjazzlab ?       FYI JJazzLab embeds its own JDK/

**Reply by dhomi** [reply]
Downloaded JJazzLab. cd to that folder. Then from there I run in terminal (zsh): bin/jjazzlab
Then I get this error: Cannot find java. Please use the --jdkhome switch.

Then I try: bin/jjazzlab --jdkhome /usr/libexec/java_home
Then I get this error: Cannot find java. Please use the --jdkhome switch.
same with: bin/jjazzlab --jdkhome "/usr/libexec/java_home" and bin/jjazzlab --jdkhome="/usr/libexec/java_home"

**Reply by jjazzboss** [reply, marked as answer]
check that all extracted files have the right permissions, especially in the jdk/bin subdir (must have execution permission for you).

**Reply by dhomi** [reply]
Yes SOLVED by your support, thanks. 

Changes needed were to make executable the jjazzlab and java like here:
chmod +x bin/jjazzlab
chmod +x jdk/bin/java

Maybe you can add this in the doc. or make a .gz for mac where these files already are executable. Zip wont remember the executable flags

**Comment by jjazzboss** [comment]
I stopped building a x64.pkg, too many problems depending on the MacOs version.
